testing/simulator/src/cli.rs

```diff
@@ -77,7 +77,6 @@ pub fn cli_app() -> Command {
                 )
                 .arg(
                     Arg::new("vc-count")
-                        .short('c')
```

Comment:
it seems we have two flags with the same short values
- continue-after-checks
- vc-count

this causes a build error so i've removed one of them

Reply:
Yep looks fine to me - I don't see any usage of the short args